io.github.narunlifescience.AlphaPlot.yml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.narunlifescience.AlphaPlot
+runtime: org.kde.Platform
+runtime-version: "5.15"
+sdk: org.kde.Sdk
+base: io.qt.qtdatavis3d.BaseApp
+#base-version: "5.15"
+command: alphaplot
+rename-icon: alphaplot
+rename-desktop-file: alphaplot.desktop
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+  # Enable Wayland but keep X11 default for now
+  - --socket=wayland
+  - --env=QT_QPA_PLATFORM=xcb
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+- name: gsl
```

Comment:
You probably want to clean up `/include` and `*.a` too.

Reply:
Updated "cleanup"